docker/helpers/check-ports.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Validate the required ports are available for the application
+function check_ports() {
```

Comment:
I see this function checks if another container has the same port. The check is good because it will prevent collation with an existing version of Fonoster. However, it does not consider other processes using a given port.

Reply:
🤔